### PR TITLE
[fix](memory) `TabletSchema` and `Schema` no longer track memory, only track columns count.

### PR DESCRIPTION
**Pull request by xinyiZzz**
## Proposed changes

1. `TabletSchema` and `Schema` no longer track memory, only track columns count. because cannot accurately track memory size.

2. TabletMeta MemTracker changed to track TabletSchema columns count.

3. Segment::_meta_mem_usage Unknown value overflow, causes the value of SegmentMeta MemTracker is similar to `-2912341218700198079`. So, temporarily put it in experimental type tracker. 
 
## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by xinyiZzz** [comment]
run buildall

**Comment by xinyiZzz** [comment]
run buildall

File: be/src/olap/tablet_manager.cpp [hunk 1]
```diff
@@ -236,10 +234,12 @@ Status TabletManager::_add_tablet_to_map_unlocked(TTabletId tablet_id,
     tablet_map_t& tablet_map = _get_tablet_map(tablet_id);
     tablet_map[tablet_id] = tablet;
     _add_tablet_to_partition(tablet);
+    // TODO: memory size of TabletSchema cannot be accurately tracked.
     // TODO: remove multiply 2 of tablet meta mem size
     // Because table schema will copy in tablet, there will be double mem cost
     // so here multiply 2
-    _tablet_meta_mem_tracker->consume(tablet->tablet_meta()->mem_size() * 2);
+    // _tablet_meta_schema_columns_tracker->consume(tablet->tablet_meta()->mem_size() * 2);
+    _tablet_meta_schema_columns_tracker->consume(tablet->tablet_meta()->tablet_columns_num());
```
Comment (yiguolei) [inline be/src/olap/tablet_manager.cpp:242]: 这既然不是一个memtracker， 就不要用memtracker 来统计个数了，后面会翻车。
如果他只是一个counter，那么就用类似bvar或者doris metric 这种方式直接作为监控项来统计。

File: be/src/olap/tablet_meta.cpp [hunk 2]
```diff
@@ -679,7 +679,7 @@ void TabletMeta::to_meta_pb(TabletMetaPB* tablet_meta_pb) {
             time_series_compaction_time_threshold_seconds());
 }
 
-uint32_t TabletMeta::mem_size() const {
+int64_t TabletMeta::mem_size() const {
```
Comment (yiguolei) [inline be/src/olap/tablet_meta.cpp:682]: 如果没有用的话，直接把这个代码删了吧

Comment (xinyiZzz) [reply be/src/olap/tablet_meta.cpp:682]: 先记着TODO留着吧，假以时日，还是要把元数据内存统计准的

File: be/src/olap/tablet_manager.cpp (hunk 1 above)
Comment (xinyiZzz) [reply be/src/olap/tablet_manager.cpp]: done，改成了bvar

**Comment by xinyiZzz** [comment]
run buildall

**Review by yiguolei — APPROVED** [review]
LGTM

**Review by Lchangliang — APPROVED** [review]

**Comment by Lchangliang** [comment]
LGTM

**Pull request merged**